### PR TITLE
update broadcastjob readme

### PR DESCRIPTION
**Pull request by jian-he**
### Ⅰ. Describe what this PR does
update broadcastjob readme

### CODE REVIEW

File: docs/concepts/broadcastJob/README.md [hunk 1]
```diff
@@ -122,9 +122,9 @@ spec:
     activeDeadlineSeconds: 10
 ```
 
-#### Use `Never` completionPolicy
+#### Use `Never` completionPolicy to always keep the job active to automatically trigger pods on newly coming nodes
```
Comment (xiang90) [inline docs/concepts/broadcastJob/README.md]: Is this better?

```
Automatically launch pods on newly coming nodes by keeping the job active using xxx
```

**Comment by xiang90** [comment]
LGTM. Thanks.

File: docs/concepts/broadcastJob/README.md (hunk 1 above)
Comment (jian-he) [reply docs/concepts/broadcastJob/README.md]: fixed

File: docs/concepts/broadcastJob/README.md [hunk 2]
```diff
@@ -122,9 +122,9 @@ spec:
     activeDeadlineSeconds: 10
 ```
 
-#### Use `Never` completionPolicy
+#### Automatically launch pods on newly coming nodes by keeping the job active using `Never` completionPolicy
 Run a BroadcastJob with `Never` completionPolicy. The job will continue to run even if all Pods 
-have completed on all nodes.
+have completed on all nodes. This is useful for automatically triggering new pods run on newly coming nodes.
```
Comment (Fei-Guo) [inline docs/concepts/broadcastJob/README.md]: ->This is useful for automatically running Pods on newly added nodes.

Comment (jian-he) [reply docs/concepts/broadcastJob/README.md]: fixed

**Pull request merged**